staging/publishing/rules.yaml

```diff
@@ -130,6 +130,10 @@ rules:
     dependencies:
     - repository: apimachinery
       branch: release-1.16
+    - repository: api
```

Comment:
didn't we have a CI verify script for that?

Reply:
It only looks for the master branch right now (will look into fixing that as a follow-up, because it isn't so clean to check that in the current python script)

Reply:
Just realized this won't also be as simple as checking out the `release-1.16` branch and verifying deps for staging repos because it won't catch the case when the `release-1.16` branch is fast-forwaded (current case) :confused: